ionoscloud/resource_server_test.go

```diff
@@ -216,6 +216,45 @@ func TestAccServerBasic(t *testing.T) {
 	})
 }
 
+func TestAccServerNoBootVolumeBasic(t *testing.T) {
```

Comment:
@gabi-ionos this test has nothing to do with [this](https://github.com/ionos-cloud/terraform-provider-ionoscloud/issues/379) issue, here you are only checking the creation of a server with a volume that it's not a boot volume (like we do for other tests), the only visible difference is that you added `delete_volumes = true` which has no effect on the creation side, so this test is useless for the issue's context.

Reply:
@cristiGuranIonos I see that the tests were modified, I also modified the function that checks if the servers/volumes are deleted properly. I did this in this commit: fa6afaad843f8bf9dbd6f1573b1fd37bc3272a79